specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/transaction.json

```diff
@@ -32,28 +376,183 @@
           },
           {
             "$ref": "./types.json#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "./types.json#/parameters/filter"
+          },
+          {
+            "$ref": "./types.json#/parameters/orderBy"
+          },
+          {
+            "$ref": "./types.json#/parameters/top"
+          },
+          {
+            "$ref": "./types.json#/parameters/skip"
+          },
+          {
+            "$ref": "./types.json#/parameters/count"
+          },
+          {
+            "$ref": "./types.json#/parameters/search"
           }
         ],
         "responses": {
           "200": {
-            "description": "OK. The request has succeeded.",
+            "description": "The list of transactions.",
             "schema": {
               "$ref": "#/definitions/TransactionListResult"
             }
           },
           "default": {
             "description": "Error response describing why the operation failed.",
             "schema": {
-              "$ref": "./types.json#/definitions/ArmError"
+              "$ref": "./types.json#/definitions/ArmErrorResponse"
             }
           }
         },
+        "x-ms-examples": {
+          "TransactionsListByInvoice": {
+            "$ref": "./examples/transactionsListByInvoice.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/invoices/{invoiceName}/transactionSummary": {
```

Comment:
Should this be `/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/invoices/{invoiceName}/transactionSummary/default` as it is not a "List"?